fedimint-testing/src/fixtures.rs

```diff
@@ -134,29 +126,44 @@ impl Fixtures {
         .await
     }
 
-    /// Starts a new gateway
-    pub async fn new_gateway(&self, password: Option<String>) -> GatewayTest {
+    /// Starts a new gateway with a given lightning node
+    pub async fn new_gateway(&self, ln: Arc<dyn LightningTest>) -> GatewayTest {
         // TODO: Make construction easier
```

Comment:
This probably isn't a TODO anymore right? Not sure exactly what its referring to now